### ISSUE TITLE
x:Static without Source parameter doesn't allow specifying converter

### ISSUE THREAD

**Issue by Tommigun1980** [xamarin/Xamarin.Forms#11991]
You can't do

```xaml
{x:Static SomethingStatic, Converter={StaticResource someConverter}}
```

but you can do

```xaml
{Binding Source={x:Static SomethingStatic}, Converter={StaticResource someConverter}}
```

It would be nice if you could use the "normal" binding syntax with static constructs as well as it gets hard to remember the special cases. I don't know if this is a bug or a feature request.

Thanks.

**Comment by StephaneDelcroix** [comment]
> I don't know if this is a bug or a feature request.

none, unfortunately. I don't see the point of overloading the x:Static markup, and make it works like a binding, without calling it a binding

**Comment by Tommigun1980** [comment]
> > I don't know if this is a bug or a feature request.
> 
> none, unfortunately. I don't see the point of overloading the x:Static markup, and make it works like a binding, without calling it a binding

Fair enough. My point was that `{x:Static SomethingStatic}` is already valid, so if it's been extended that way why not also allow a converter parameter?
